src/Microsoft.AspNetCore.Razor.Tools/project.json

```diff
@@ -1,11 +1,17 @@
 {
   "version": "1.0.0-*",
-  "description": "TagHelper tooling for .NET Core CLI.",
+  "description": "TagHelper tooling for .NET Core CLI. Contains an executable to resolve TagHelperDescriptors for a given project.",
   "packOptions": {
     "repository": {
       "type": "git",
       "url": "git://github.com/AspNet/RazorTooling"
-    }
+    },
+    "tags": [
+      "dotnet-razor-tooling",
```

Comment:
This tag isn't saying much... If anything, perhaps say the command line tool name in the description. E.g. `"Contains the dotnet-razor-tooling command used to resolve XYZ for projects at design-time."`.